lib/tasks/move_discontinued_at_to_services.rake

```diff
@@ -0,0 +1,19 @@
+namespace :services do
+  desc "Move discontinued at from discontinued_services to services"
+  task move_discontinued_at_to_services: :environment do
+    services = Service.all
+    puts "Going to update each service discontinued at"
+
```

Comment:
A rake task to move `discontinued_at` from `discontinuedservice` to service.

Reply:
and not removing the `discontinued service` model and drop its table until this runs because you need the `has_many` relationship to move data.

Reply:
Looks correct to me!